commit/merkleroot/rmn/controller.go

```diff
@@ -36,7 +36,7 @@ var (
 
 	// ErrNothingToDo is returned when there are no source chains with enough RMN nodes.
 	ErrNothingToDo = errors.New("nothing to observe from the existing RMN nodes, make " +
-		"sure RMN is enabled, nodes configured correctly and minObservers value is correct")
+		"sure RMN is enabled, nodes configured correctly and F value is correct")
```

Comment:
Lets be consistent with the casing of F - upper case or lower case, lets stick to one of them.